### PR TITLE
docs: update the mac and linux setup doc

### PR DESCRIPTION
**Pull request by reidliu41**
there is a small change for the doc

### CODE REVIEW

File: docs/mac_development.md [hunk 1]
```diff
@@ -63,6 +69,7 @@ You can build an image with your changes and use that image in docker-compose
 2. export EDA_IMAGE=localhost/myserver:latest
 3. docker-compose -f ./tools/docker/docker-compose-mac.yml up
 
+You can now access the UI at <https://localhost:8443/eda/> with default login username and password(admin/testpass).
```
Comment (Alex-Izquierdo) [inline docs/mac_development.md:72]: Would you mind to add the same line for linux as well?

File: docs/mac_development.md [hunk 2]
```diff
@@ -44,7 +44,13 @@ podman-machine-default ssh://core@localhost:49473/run/user/501/podman/podman.soc
 ```
 example use the uid 501 above and port 49473 from your enviornment in the next step
 
-2. ssh -fnNT -L/tmp/podman.sock:/run/user/{uid}/podman/podman.sock -i ~/.ssh/podman-machine-default ssh://core@localhost:{port} -o StreamLocalBindUnlink=yes
+2. Create a secure tunnel, and forward the local socket to the remote socket 
+```
+RUN_USER_ID=501
+REMOTE_SSH_PORT=49473
```
Comment (Alex-Izquierdo) [inline docs/mac_development.md:50]: Why are these lines necessary? Isn't there an "export" missing in front or be present in the same line as the `ssh` command? people from @ansible/eda-maintainers with apple could verify this instructions.

File: docs/mac_development.md (hunk 1 above)
Comment (reidliu41) [reply docs/mac_development.md:71]: sure, added

**Review by Alex-Izquierdo — APPROVED** [review]
LGTM

**Review by ttuffin — APPROVED** [review]
